### PR TITLE
LKE-7473 fix missing error in createSourceConfig

### PR DESCRIPTION
**Pull request by davidrapin**
related to:
- frontend PR: https://github.com/Linkurious/linkurious-frontend/pull/4596
- https://linkurious.atlassian.net/browse/LKE-7473

fix
- fix missing error in createSourceConfig 
- auto linting

### CODE REVIEW

**Comment by davidrapin** [comment]
@mzorkany review welcome :)

**Comment by gmasclet** [comment]
@davidrapin I guess this fix is related to https://linkurious.atlassian.net/browse/LKE-7473

Shall we also change the return type of the `SourcesService.create()` method accordingly? See https://github.com/Linkurious/linkurious-frontend/blob/96858d140f834e107b5f74705a5d3ed92bed782c/src/app/services/sources/index.ts#L116-L130

**Comment by davidrapin** [comment]
@gmasclet yes, I'm going so in https://github.com/Linkurious/linkurious-frontend/pull/4596

File: src/api/DataSource/index.ts [hunk 1]
```diff
@@ -162,7 +163,7 @@ export class DataSourceAPI extends Request {
    */
   public createSourceConfig(this: Request<number>, params: SelectedDataSourceConfig) {
     return this.request({
-      errors: [UNAUTHORIZED, FORBIDDEN],
+      errors: [UNAUTHORIZED, FORBIDDEN, INVALID_PARAMETER],
```
Comment (mzorkany) [inline src/api/DataSource/index.ts:166]: I was checking the server code, I see that when we create a source config we try to connect as well

https://github.com/Linkurious/linkurious-server/blob/b644710cf6f06fb060a1235dcb97df6c49fbfd42/server/services/data/index.ts#L502

and inside the connect function we reject with a `DATA_SOURCE_UNAVAILABLE` error if we fail to connect

https://github.com/Linkurious/linkurious-server/blob/b644710cf6f06fb060a1235dcb97df6c49fbfd42/server/services/data/dataSource.ts#L749

so should we add the `DATA_SOURCE_UNAVAILABLE` to the list of expected errors ?

**Review by mzorkany — COMMENTED** [review]
Review done :) 
I left one comment regarding the `DATA_SOURCE_UNAVAILABLE` error that we might need to add to the list of possible errors as well

**Review by mzorkany — APPROVED** [review]

**Pull request merged**